addons/account/views/account_payment_view.xml

```diff
@@ -312,7 +312,6 @@
                             </group>
                         </group>
                     </sheet>
-                    <div class="o_attachment_preview"/>
```

Comment:
Are you removing a feature in a stable version? :laughing: 
on 16.0:
![image](https://user-images.githubusercontent.com/21679159/210753302-3a558b6d-cdc7-4529-a823-d0e932d1c0e5.png)

on your branch:
![image](https://user-images.githubusercontent.com/21679159/210753556-9e1ec656-6c53-4c21-b30a-6890bbf0203e.png)

Reply:
Hmm I'll check again, as you can see I retargeted from master and was expecting the same behaviour.
In master, I had the preview there even without this node, as another code added it.
(And was able to have the preview showing twice)